x-pack/plugins/observability_solution/asset_manager/common/constants_entities.ts

```diff
@@ -11,3 +11,4 @@ export const ENTITY_TRANSFORM_PREFIX = `entities-observability-summary-${ENTITY_
 export const ENTITY_DEFAULT_FREQUENCY = '1m';
 export const ENTITY_DEFAULT_SYNC_DELAY = '60s';
 export const ENTITY_API_PREFIX = '/api/entities';
+export const ENTITY_INTERNAL_API_PREFIX = '/internal/entities';
```

Comment:
```suggestion
export const ENTITY_INTERNAL_API_PREFIX = '/internal/api/entities';
```

Reply:
Is there a reason why these are internal only? This would prevent SREs from automating the setup via Teraform.

Reply:
all apis are internal for the initial release